### PR TITLE
address some editorial issues raised by Ben Kaduk

### PR DESCRIPTION
**Pull request by stpeter**

### CODE REVIEW

File: BCP195bis/draft-ietf-uta-rfc7525bis.md [hunk 1]
```diff
@@ -525,8 +524,11 @@ Cryptographic algorithms weaken over time as cryptanalysis improves: algorithms
                encryption (which provide 40 or 56 bits of security).
 
   Rationale: Based on {{!RFC3766}}, at least 112 bits 
-               of security is needed.  40-bit and 56-bit security are considered 
-               insecure today.  TLS 1.2 never negotiates 40-bit or 56-bit export ciphers and such ciphers are not supported at all in TLS 1.3.
+               of security is needed.  40-bit and 56-bit security (found in 
+               so-called "export ciphers") are considered 
+               insecure today.  Although such ciphers are allowed under TLS 1.2, 
+               implementations that conform to {{!RFC5246}} would never negotiate 
+               such ciphers, and they are not supported at all in TLS 1.3.
```
Comment (yaronf) [inline BCP195bis/draft-ietf-uta-rfc7525bis.md:531]: I'm not following the logic here, e.g. how do we tie RFC5246 to TLS 1.3.

File: BCP195bis/draft-ietf-uta-rfc7525bis.md [hunk 2]
```diff
@@ -696,7 +698,7 @@ Rationale: the extension does not apply to the AEAD
 The recommendations of this document primarily apply to the implementation and deployment of application protocols that are most commonly used with TLS and DTLS on the Internet today.  Examples include, but are not limited to:
 
 
-* Web software and services that wish to protect HTTP traffic with TLS.
+* Web software and services that wish to protect HTTP traffic with TLS. (HTTP implementations not intended for use in the World Wide Web, e.g. minimal implemenations deployed only on constrained devices, are not necessarily included in this applicability statement.)
```
Comment (yaronf) [inline BCP195bis/draft-ietf-uta-rfc7525bis.md:701]: This is a significant exception, I don't think it belongs in parenthesis and maybe it should get its own bullet.

**Review by yaronf — CHANGES_REQUESTED** [review]

File: BCP195bis/draft-ietf-uta-rfc7525bis.md [hunk 3]
```diff
@@ -525,8 +524,11 @@ Cryptographic algorithms weaken over time as cryptanalysis improves: algorithms
                encryption (which provide 40 or 56 bits of security).
 
   Rationale: Based on {{!RFC3766}}, at least 112 bits 
-               of security is needed.  40-bit and 56-bit security are considered 
-               insecure today.  TLS 1.2 never negotiates 40-bit or 56-bit export ciphers and such ciphers are not supported at all in TLS 1.3.
+               of security is needed.  40-bit and 56-bit security (found in 
+               so-called "export ciphers") are considered 
+               insecure today.  Although such ciphers are allowed under TLS 1.2,
```
Comment (thomas-fossati) [inline BCP195bis/draft-ietf-uta-rfc7525bis.md:529]: Not sure _"allowed"_ is the right word.  Export ciphers are already "MUST NOT negotiate" in TLS 1.1 (see https://www.rfc-editor.org/rfc/rfc4346#appendix-A.5), and have been kept alive only for when version negotiation selects 1.0 or earlier.

**Comment by yaronf** [comment]
@stpeter Please add a comment to the PR of the form "Addresses #xxx, #yyy" that will ensure these issues are closed when the PR is merged.

File: BCP195bis/draft-ietf-uta-rfc7525bis.md (hunk 1 above)
Comment (stpeter) [reply BCP195bis/draft-ietf-uta-rfc7525bis.md:531]: Remove "      TLS 1.2 never negotiates 40-bit or 56-bit export ciphers and such
      ciphers are not supported at all in TLS 1.3."

**Pull request merged**